models/cow_protocol/ethereum/cow_protocol_ethereum_schema.yml

```diff
@@ -136,6 +136,9 @@ models:
       - &order_type
         name: order_type
         description: "SELL/BUY string indicating if the trade was a sell or buy order"
+      - &partial_fill
+        name: partial_fill
+        description: "Boolean indicating if the order was partially filled"
```

Comment:
I add the new column in the schema, the description may not be correct.

Reply:
So close, Partially "fillable"

Reply:
We've named it `partial_fill` in our SQL code.

Reply:
perhaps this is more clear.

```suggestion
      - &partial_fill
        name: partial_fill
        description: "Boolean indicating if the order is partially fill-able"
```

Reply:
Indeed it is. Thanks Ben!